### PR TITLE
Correct grammar mistakes

### PR DESCRIPTION
**Pull request by Hiroko103**

### CODE REVIEW

File: TS3AudioBot/CommandSystem/Commands/FunctionCommand.cs [hunk 1]
```diff
@@ -246,7 +246,7 @@ private static object GetDefault(Type type)
 		}
 
 		/// <summary>
-		/// A conveniance method to set the amount of required parameters and returns this object.
```
Comment (Splamy) [inline TS3AudioBot/CommandSystem/Commands/FunctionCommand.cs]: i think `convenience` was the word i was looking here for.

**Comment by Splamy** [comment]
Holy moly, I didn't expect this project to have *that* many grammar mistakes. Really nice work here, I have added a comment to one change but apart from that, looks good.

File: TS3AudioBot/CommandSystem/Commands/FunctionCommand.cs (hunk 1 above)
Comment (Hiroko103) [reply TS3AudioBot/CommandSystem/Commands/FunctionCommand.cs]: Oh, I see. I didn't know it was an actual term.

**Comment by Hiroko103** [comment]
Thanks! Nice to see an awesome project like this.

**Pull request merged**